gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-milestone-3-bin.zip
```

- [ ] Update the Gradle version when Gradle 7 is released

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.0-bin.zip
```

---

Will be done in a separate ticket later